### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rdlester**
* List Docker as primary installation method
* Link to installing Docker and Docker tutorials
* Describe provided Docker containers, and offer a bit of guidance on how to do stuff with them

### CODE REVIEW

**Comment by rdlester** [comment]
Fixes #99

File: README.md [hunk 1]
```diff
@@ -1,77 +1,31 @@
...
 
-OR you can connect to container over 3306 using the local mysql client if you
-have it installed. See [this article](http://stackoverflow.com/a/32361238/103315)
-on why you need to use `127.0.0.1`.
+`docker-compose up` will install and launch all our containers.  These containers are:
 
-```bash
-mysql -h 127.0.0.1 -P 3306 -u deltanyc -ppassword deltanyc
-```
+* db: The MySQL database storing NYC city data both in semi-raw form and joined into a model for the Rails website. The database can be examined with `docker exec deltanycheatseek_db_1 mysql -u <user from .env> -p <password from .env>`. Local clients can connect at 127.0.0.1:3306 ([localhost will not work](http://stackoverflow.com/a/32361238/103315)).
```
Comment (chrishenry) [inline README.md:20]: Nice. 

You can also use the shorter service name of the container as well. So `docker exec db mysql -u ...`

File: README.md [hunk 2]
```diff
@@ -1,77 +1,31 @@
...
+* web: Installs Ruby on Rails and starts a server at [localhost:3000](http://localhost:3000). Like the data-import scripts, you can conect to a running instance with `docker exec -it deltanycheattseek_web_1 /bin/bash` to run rake tasks. To view a demo of the tool, connect to the instance, then run `rake db:setup`. In the demo, fake data will be provided for "10 West 109th Street".
 
-A note on preg UDFs
-***************************
-The cleanup utils make extensive use of [User Defined preg functions](https://github.com/mysqludf/lib_mysqludf_preg).
-These aren't available in the general MySQL docker container, and the container
-provided by the [uqlibrary](https://github.com/uqlibrary/docker-mysql-udf-preg)
-hasn't been built in a while, so it doesn't have recent fixes to the upstream
-MySQL container.
+Once everything is installed, only the nb or web containers need to be explicitly lanched (eg. `docker-compose up web`) depending on the type of work you plan to do. Launching either will also launch the db container.
```
Comment (chrishenry) [inline README.md]: I don't quite follow what you mean by `installed` here. Do you mean the Docker engine? Or are you referring to the `docker-compose up` from just above?

File: README.md (hunk 1 above)
Comment (rdlester) [reply README.md:20]: That's what you originally had written in the README, but I've never been able to get it to work.
For example, `docker exec db /bin/bash` (or even with deltanycheatseek_db) gets:
"Error response from daemon: No such container: db".

File: README.md (hunk 2 above)
Comment (rdlester) [reply README.md]: I mean `docker-compose up`: ie the individual containers / images are all installed. The language is a little clunky. Is there better terminology for Docker? Any language suggestions?

File: README.md (hunk 1 above)
Comment (chrishenry) [reply README.md:20]: Ohhh, I see why. It should be `docker-compose`, not `docker`. 

Sorry about that... 😦

File: README.md [hunk 3]
```diff
@@ -1,77 +1,31 @@
...
+* web: Installs Ruby on Rails and starts a server at [localhost:3000](http://localhost:3000). Like the data-import scripts, you can conect to a running instance with `docker exec -it deltanycheattseek_web_1 /bin/bash` to run rake tasks. To view a demo of the tool, connect to the instance, then run `rake db:setup`. In the demo, fake data will be provided for "10 West 109th Street".
 
-A note on preg UDFs
-***************************
-The cleanup utils make extensive use of [User Defined preg functions](https://github.com/mysqludf/lib_mysqludf_preg).
-These aren't available in the general MySQL docker container, and the container
-provided by the [uqlibrary](https://github.com/uqlibrary/docker-mysql-udf-preg)
-hasn't been built in a while, so it doesn't have recent fixes to the upstream
-MySQL container.
+Once all the images are installed and run once, only the nb or web images need to be explicitly lanched via docker-compose (eg. `docker-compose up web`) depending on the type of work you plan to do. Launching either will also launch the db container.
```
Comment (chrishenry) [inline README.md]: This is definitely closer. The only other thing I would change is `installed` to `built`. `docker-compose up` will build containers as part of `up`.

Comment (rdlester) [reply README.md]: done

**Pull request merged**